frontend/pages/dashboard/index.tsx

```diff
@@ -1,3 +1,15 @@
+import DashboardHeader from '../../compositions/dashboardHeader/index';
+import styles  from '../../compositions/dashboardHeader/header.module.css';
```

Comment:
I've always been warned not to apply other components' local styles to other components - seems like `backgroundBanner` might best live inside the `DashboardHeader` component directly?

Reply:
Also I think we'll want to rename the css file to match the new component name!

Reply:
Thanks for the catch - I've removed background banner from the dashboard component all together, and lives inside DashboardHeader directly. CSS file renamed as well!